ImageRow.xcodeproj/project.pbxproj

```diff
@@ -292,7 +392,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.3;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```

Comment:
I don't think we want to increase this. I think this is a suggestion from Xcode but ImageRow should be fine prior to iOS 12

Reply:
@mats-claassen I have updated my code to revert back to 9.3